lib/libraries/aws_object_upload_file_wrapper.rb

```diff
@@ -0,0 +1,27 @@
+require "aws-sdk-s3"
+
+# Wraps Amazon S3 object actions.
+class AwsObjectUploadFileWrapper
+  attr_reader :object
+
+  # @param object [Aws::S3::Object] An existing Amazon S3 object.
+  def initialize(file_path)
+    bucket_name = Figaro.env.AWS_S3_BUCKET_NAME
+    object_key = "#{Figaro.env.AWS_S3_PATH}#{File.basename(file_path)}"
```

Comment:
Just out of curiosity, does this require that `AWS_S3_PATH` end (or not end) in a slash? If it's fragile like that, then can we use something like `File.join` to compose the path, to avoid that fragility?

If it's not a problem, disregard.

Reply:
good point, done